src/lesson6_task1/AirTransport.java

```diff
@@ -0,0 +1,18 @@
+package lesson6_task1;
+
+public abstract class AirTransport extends Transport {
+    float wingspan;
+    int minRunwayLength;
+
+    //конструктор
+    public AirTransport(int powerHours, int maxSpeed, String brand, float weight,   //унаследование поля
+                        float wingspan, int minRunwayLength) {              // поля экзепляров етого класа
```

Comment:
@Nadya101  не критично, улыбнуло, но стоит грамматику даже в коментариях проверять☺☺
```suggestion
                        float wingspan, int minRunwayLength) {              // поля экзепляров Этого класа
```

Reply:
Исправлено &#129318;